docs/services/navigation/_index.md

```diff
@@ -118,7 +134,7 @@ The following attributes are available for `Navigation` services:
 | `base` | string | **Required** | The `name` you have configured for the [base](/components/base/) you are operating with this service. |
 | `movement_sensor` | string | **Required** | The `name` of the [movement sensor](/components/movement-sensor/) you have configured for the base you are operating with this service. |
 | `motion_service` | string | Optional | The `name` of the [motion service](/services/motion/) you have configured for the base you are operating with this service. If you have not added a motion service to your robot, the default motion service will be used. Reference this default service in your code with the name `"builtin"`. |
-| `vision_services` | array | Optional | The `name` of each [vision service](/services/motion/) you have configured for the base you are operating with this service. |
+| `obstacle_detectors` | array | Optional | An array containing objects with the `name` of each [`"camera"`](/components/camera/) you have configured for the base you are navigating, along with the `name` of the [`"vision_service"`](/services/motion/) you are using to detect obstacles. |
```

Comment:
[nit]: not sure where this would go but, could be useful.

Lets say you have a robot composed a main part and a remote part.

If vision svc A is on a remote, and camera B is on the main part, then A would not be able to see B.
Here vision svc A only has access to other remote cameras.

If vision svc C is on the main part, it has access to cameras on the main part but also the remote part.

Reply:
```suggestion
| `obstacle_detectors` | array | Optional | An array containing objects with the `name` of each [`"camera"`](/components/camera/) you have configured for the base you are navigating, along with the `name` of the [`"vision_service"`](/services/motion/) you are using to detect obstacles. Note that any vision services on remote parts will only be able to access cameras on the same remote part. |
```
@nfranczak Hmm--would this be helpful/accurate?

Reply:
yep!